conda/core/solve.py

```diff
@@ -397,6 +400,9 @@ def solve_for_diff(self, deps_modifier=NULL, prune=NULL, ignore_pinned=NULL,
         # TODO: force_reinstall might not yet be fully implemented in :meth:`solve_final_state`,
         #       at least as described in the docstring.
 
+        unlink_precs = IndexedSet(reversed(sorted(unlink_precs,
```

Comment:
With this, can the following comment be removed?:
```
         # TODO: get the sort order correct for unlink_records
```

Reply:
Yes, I'll remove it in that follow-on PR though that I spoke with you about.  Would rather not run CI again for this PR.

Reply:
What is the follow-on PR?

Reply:
#6394